### PR TITLE
Add local build script

### PR DESCRIPTION
**Pull request by SharonHart**
Add a script starting the cluster with compose, then running the tests.

### CODE REVIEW

**Comment by navalev** [comment]
the dockerfile build is timely (due to using the L version of spacy) - can we use the S version of spacy for the local build?

File: run.bat [hunk 1]
```diff
@@ -0,0 +1,28 @@
+echo >/dev/null # >nul & GOTO WINDOWS & rem ^
+
+export REGISTRY_NAME=
```
Comment (itye-msft) [inline run.bat:3]: Nice. I am missing instructions to populate the optional params.

**Review by itye-msft — CHANGES_REQUESTED** [review]

File: run.bat (hunk 1 above)
Comment (SharonHart) [reply run.bat:3]: The export I'm doing here?
Those are just so that warnings won't show when running compose.
The REGISTRY_NAME, IMAGE_PREFIX, TAG are for the CI process and should be changed during local build.

**Comment by SharonHart** [comment]
@navalev It should match the CI default model, since e2e tests might have static scores on the assertion that depend on the model size. Omri insisted on lg throughout all the project as far as I remember. Will add a comment on the times

**Review by itye-msft — APPROVED** [review]

File: .env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+REGISTRY_NAME=''
```
Comment (navalev) [inline .env:1]: why is this file needed? seems to fail the tests

Comment (SharonHart) [reply .env:1]: It's default env cars used in compose. They are used when pushing images so locally should default to empty string.
Adding this allowed removing exports in the script, and not printing warning when running compose.
It's should not fail the build, I think it fails from the sporadic issue

**Review by navalev — APPROVED** [review]
tested and pass on windows

**Review by itye-msft — APPROVED** [review]
